src/Sylius/Component/Core/Repository/ProductRepositoryInterface.php

```diff
@@ -34,10 +34,17 @@ public function createListQueryBuilder(string $locale, $taxonId = null): QueryBu
      * @param TaxonInterface $taxon
      * @param string $locale
      * @param array $sorting
+     * @param bool $reproduceTaxonomyTree
      *
      * @return QueryBuilder
      */
-    public function createShopListQueryBuilder(ChannelInterface $channel, TaxonInterface $taxon, string $locale, array $sorting = []): QueryBuilder;
+    public function createShopListQueryBuilder(
+        ChannelInterface $channel,
+        TaxonInterface $taxon,
+        string $locale,
+        array $sorting = [],
+        bool $reproduceTaxonomyTree = false
```

Comment:
Btw. I wouldn't get what it does just by that variable, `$includeAllDescendants` or sth else?